handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -3320,10 +3315,15 @@ export default () => {
      * | `samplingRatio`         | A number                        | The number of samples of the same length to be used in column width calculations                               |
      * | `allowSampleDuplicates` | `true` \| `false`               | When calculating column widths:<br>`true`: Allow duplicate samples<br>`false`: Don't allow duplicate samples |
      *
+     * By default, the `autoColumnSize` option is set to `undefined`,
+     * but the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin acts as enabled.
+     * To disable the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin completely,
+     * set the `autoColumnSize` option to `false`.
+     *
      * Using the [`colWidths`](#colWidths) option forcibly disables the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin.
      *
      * Read more:
-     * - [`AutoColumnSize`](@/api/autoColumnSize.md)
+     * - [`AutoColumnSize` &#8594;](@/api/autoColumnSize.md)
```

Comment:
Is this change intentional?

Reply:
@wszymanski - yup, a minor thing I spotted and wanted to fix straight away